### PR TITLE
Additional package metadata checks

### PR DESCRIPTION
**Pull request by MarekSuchanek**
Added few more asserts after recalling what was told during the tutorials (have correct version, include name and some keywords - but still classifiers are more important)...

### CODE REVIEW

File: tests_module/test_distribution.py [hunk 1]
```diff
@@ -93,12 +93,22 @@ def test_package_info(utils, tmpdir, sh):
...
+                values[m] = item[1]
                 found = True
                 assert len(item[1]) > 0, 'Metadata {} is empty'.format(m)
                 break
         assert found, 'Metadata {} is not present'.format(m)
+
+    # Check name, version, keywords as @hroncok mentioned during tutorials
+    assert values['Name'] == utils.package_name.replace('_', '-'), \
+        'Bad package name in metadata'
+    assert values['Version'] == '0.3', \
```
Comment (hroncok) [inline tests_module/test_distribution.py]: I'd go with startswith here maybe

Comment (MarekSuchanek) [reply tests_module/test_distribution.py]: good point

**Review by hroncok — APPROVED** [review]
Works for me

File: tests_module/test_distribution.py [hunk 2]
```diff
@@ -93,12 +93,22 @@ def test_package_info(utils, tmpdir, sh):
...
                 assert len(item[1]) > 0, 'Metadata {} is empty'.format(m)
                 break
         assert found, 'Metadata {} is not present'.format(m)
+
+    # Check name, version, keywords as @hroncok mentioned during tutorials
+    assert values['Name'] == utils.package_name.replace('_', '-'), \
+        'Bad package name in metadata'
+    assert values['Version'].startswith('0.3'), \
+        'Bad package version in metadata'
+    assert len(values['Keywords'].split(' ')) > 3, \
```
Comment (hroncok) [inline tests_module/test_distribution.py]: actually, can be split by spaces or commas.

**Review by hroncok — CHANGES_REQUESTED** [review]

File: tests_module/test_distribution.py (hunk 2 above)
Comment (MarekSuchanek) [reply tests_module/test_distribution.py]: :disappointed: ok

**Pull request merged**